static/js/pages/instances/InstanceTerminal.tsx

```diff
@@ -135,6 +135,8 @@ const InstanceTerminal: FC<Props> = ({ controlTarget, notify }) => {
       return;
     }
 
+    xtermRef.current?.terminal.element?.style.setProperty("padding", "1rem");
```

Comment:
Isn't it better to add a class with this style instead of setting it inline?

Reply:
It messes up the fit addon when done in another way. so the cols/rows won't match actual screen sice.